addon/components/frost-select-dropdown.js

```diff
@@ -91,7 +91,7 @@ export default Component.extend({
     ]
       .join(';')
 
-    return Ember.String.htmlSafe(style)
+    return EmberString.htmlSafe(style)
```

Comment:
Alright, then why not just leave the invocation as `Ember.String.htmlSafe()` instead of trying to destructure this and having to assign it to an alias.